tests/compiletest.rs

```diff
@@ -153,28 +118,20 @@ fn get_host() -> String {
     version_meta.host
 }
 
-fn run_pass_miri(opt: bool) {
-    let sysroot = get_sysroot();
-    let host = get_host();
+fn get_target() -> String {
+    std::env::var("MIRI_TARGET").unwrap_or_else(|_| get_host())
+}
 
-    for_all_targets(&sysroot, |target| {
```

Comment:
with this missing, CI isn't testing all the targets anymore that it was testing before

Reply:
CI is testing exactly the same targets as before: `.travis-ci.yml` now calls `cargo test` multiple times to do that.

Reply:
Oh. I misread what's happening, we're just testing one target other than the host. I was looking for some kind of loop